spec/PushController.spec.js

```diff
@@ -503,6 +503,62 @@ describe('PushController', () => {
       .then(done).catch(done.fail);
   });
 
+  it('properly creates _PushStatus without serverURL', (done) => {
+    const pushStatusAfterSave = {
+      handler: function() {}
+    };
+    Parse.Cloud.afterSave('_PushStatus', pushStatusAfterSave.handler);
+    const installation = new Parse.Object("_Installation");
+    installation.set("installationId", "installation");
+    installation.set("deviceToken","device_token")
+    installation.set("badge", 0);
+    installation.set("originalBadge", 0);
+    installation.set("deviceType", "ios");
+
+    var payload = {data: {
+      alert: "Hello World!",
+      badge: 1,
+    }}
+
+    var pushAdapter = {
+      send: function(body, installations) {
+        return successfulIOS(body, installations);
+      },
+      getValidPushTypes: function() {
+        return ["ios"];
+      }
+    }
+
+    var config = new Config(Parse.applicationId);
+    var auth = {
+      isMaster: true
+    }
+    var pushController = new PushController();
+    return installation.save().then(() => {
```

Comment:
why isn't setting the master key necessary?

Reply:
not sure, but, it works, you can save any installation you want